frontend/src/lib/components/PropertyKeyInfo.tsx

```diff
@@ -229,6 +229,19 @@ export const keyMapping: KeyMappingInterface = {
             label: 'Capture Failed Request',
             description: '',
         },
+        $plugins_succeeded: {
+            label: 'Plugins Succeeded',
+            description: 'Plugins that ran successfully on an event.',
+        },
+        $plugins_failed: {
+            label: 'Plugins Failed',
+            description: 'Plugins that tried to run on an event but failed.',
+        },
+        $plugins_deferred: {
+            label: 'Plugins Deferred',
+            description:
+                'Plugins that had read-only access to an event for some asynchronous processing (e.g. for exporting it).',
```

Comment:
Maybe mention which of these relate specifically to `processEvent` and which to `onEvent`/`exportEvents` – that's the crucial difference.

Reply:
I guess that's worth it - might confuse non-technical users though (or even technical non-plugin dev users)